DependencyInjection/Configuration.php

```diff
@@ -124,7 +124,11 @@ private function addDocumentManagersSection(ArrayNodeDefinition $rootNode)
                                     ->booleanNode('pretty')->defaultValue('%kernel.debug%')->end()
                                 ->end()
                             ->end()
-                            ->scalarNode('default_repository_class')->defaultValue(DocumentRepository::class)->end()
+                            ->scalarNode('default_repository_class')
+                                ->defaultValue(DocumentRepository::class)
+                                ->setDeprecated('The child node "%node%" at path "%path%" is deprecated. Please use default_document_repository_class instead.')
```

Comment:
With `setDeprecated` it's time to bump Symfony requirements, it wasn't available in 2.x and `prefer-lowest` build is failing

Reply:
Goodbye Symfony < 3.4 then.